react/components/EXPERIMENTAL_Table/BulkActions/index.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { FC } from 'react'
+import csx from 'classnames'
+
+import Button from '../../Button'
+import ButtonWithIcon from '../../ButtonWithIcon'
+import ActionMenu from '../../ActionMenu'
+import Close from '../../icon/Close'
+
+import useTableContext from '../hooks/useTableContext'
+import { ORDER_CLASSNAMES, NAMESPACES } from '../constants'
+
+const BulkActions: FC = () => {
+  const {
+    bulkState,
+    selectAllRows,
+    deselectAllRows,
+    bulkActions,
+    hasBulkActions,
+    hasPrimaryBulkAction,
+    hasSecondaryBulkActions,
+  } = useTableContext()
+
+  const selectedRowsLength = bulkState.selectedRows.length
```

Comment:
What if `bulkState` is not defined?

Reply:
`bulkState` is returned by the `useTableBulkActions` hook, so it is always defined.